dask/bytes/tests/test_local.py

```diff
@@ -70,6 +70,17 @@ def test_with_urls():
         assert sum(map(len, values)) == len(files)
 
 
+def test_with_paths():
+    pathlib = pytest.importorskip('pathlib')
+    with filetexts(files, mode='b'):
+        url = pathlib.Path('./.test.accounts.*')
+        sample, values = read_bytes(url, blocksize=None)
+        assert sum(map(len, values)) == len(files)
+    with pytest.raises(OSError):
```

Comment:
I'm wary of people using Path objects to store URLs, so it would be nice if that check was explicit.